.circleci/config.yml

```diff
@@ -206,6 +206,7 @@ jobs:
             ./scripts/ci_check_if_test_should_run_v2.sh ${FILES_TO_CHECK}
       - run:
           name: test
+          no_output_timeout: 20m
```

Comment:
Why is this change necessary?

Reply:
was increased because timeouts were found in CircleCI, I can remove it